narayana-release-process.sh

```diff
@@ -121,14 +108,6 @@ then
   echo 1>&2 Could not deploy narayana to nexus
   exit
 fi
-git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
-if [[ $? != 0 ]]
-then
-  echo 1>&2 COULD NOT BUILD Narayana RELEASE PKGS
-  exit
-fi
 cd -
 
-
-# not sure why we need to look at CI at this point so commenting it out
-# xdg-open https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/ &
+echo "Try to run the quickstarts with the release in CI"
```

Comment:
The wording here should be more inline with https://github.com/jbosstm/narayana/commit/120febc4d0c7809faedbfbcdbf0a7545fc4918bc#diff-530cdb1e8542754f0d5c3b24ac1598f52957ed33177c8ca69b719666077c3bbeR125

I don't think there will be a blog post though so maybe just "Please visit Narayana CI and check the quickstarts are working with the release and obtain performance numbers"?

Reply:
Ah, it's my branch. I will adapt!